### PR TITLE
Feature: support 2 new syntax for array of schema in @return and in @property

### PR DESCRIPTION
**Pull request by meabed**
### What kind of change does this PR introduce? (check at least one)
 - [ ] Bugfix
 - [X] Feature
 - [ ] Test
 - [ ] Docs
 - [ ] Refactor
 - [ ] Build-related changes
 - [ ] Other, please describe:

### Description:
Update packages to latest version, no breaking changes.

Ability to describe response as array of different schema example:
```
      /**
       * GET /api/v1
       * @summary This is the summary or description of the endpoint
       * @return {array<Song|Album>} 200 - success response - application/json
       */
// this allow you to have schema that match this response
res.json( [ {songId:1} , {songId:2} , {AlbumId:3} ] )
```

Ability to describe property as array of different schemas, example: 
```
    /**
     * Album
     * @typedef {object} Album
     * @property {anyOf|Song[]|Album|string|string[]} firstSong
     */
```

Convert  Null to {} in swagger

### CODE REVIEW

File: package.json
Comment (kevinccbsg) [inline package.json:12]: Hi @meabed, thanks for your PR 👏 However, we could not merge it because of this new version. This is a breaking change of the examples. We are dealing with this issue in this [PR](https://github.com/BRIKEV/express-jsdoc-swagger/pull/126) the new version breaks some of the example responses in the Swagger UI.


After we solve this we will merge this. I hope by the end of this month we could do it 🚀 🙌

Comment (meabed) [reply package.json:12]: Great, thank you!

Comment (kevinccbsg) [reply package.json:12]: Hi @meabed the PR that I mentioned was merged 👍 So if you could fix those package.lock errors we could merge this 👍 

Thanks!!!

**Review by kevinccbsg — APPROVED** [review]

**Comment by meabed** [comment]
Thank you @kevinccbsg, I have merged the upstream and the installed same packages. all looks good now.

**Comment by kevinccbsg** [comment]
@all-contributors please add @meabed for code

**Review by bri06 — APPROVED** [review]
LGTM

**Pull request merged**
